apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -135,4 +152,8 @@ export default {
   .oc-text {
     font-size: 1rem;
   }
+
+  .oc-disabled {
```

Comment:
one day we need to move this to ODS ....

Reply:
Already working on this :wink: